yarn/pom.xml

```diff
@@ -109,6 +109,16 @@
       <properties>
         <jersey.version>1.9</jersey.version>
       </properties>
+      <!-- Enable hadoop-2.2 profile when the Hadoop version is 2.2.0, which is now 
+      the default. Those building against 2.2.x or some distro-specific variation of 
+      that string would still have to manually enable -Phadoop-2.2.
+      -->
+      <activation>
+       <property>
+        <name>hadoop.version</name>
```

Comment:
What about a theoretical "2.2.1"? In other similar profiles we haven't set it up this way so I wouldn't add this purely for consistency.